Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:20
+
+COPY . .
+RUN npm i --omit=dev --no-package-lock
+USER node
+
+CMD ["node","./src/index.js"]
```

Comment:
what about `npm start`?

Reply:
I'm awaiting further input before I make the requested change  :+1: